### ISSUE TITLE
[JSON-LD WG] CR Update Request for json ld11 and json ld11 api

### ISSUE THREAD

**Issue by iherman** [w3c/transitions#230]
# Document URLs

- JSON-LD 1.1
    - Editor's draft: https://w3c.github.io/json-ld-syntax/?specStatus=CR&publishDate=2020-03-02
    - Expected publication date: 2020-03-02
- JSON-LD 1.1 Processing Algorithms and API
    - Editor's draft: https://w3c.github.io/json-ld-api/?specStatus=CR&publishDate=2020-03-02
    - Expected publication date: 2020-02-02

# Link to group's decision to request transition

https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2020/2020-01-17-json-ld#resolution3

# Link to previous Candidate Recommendation transition request

See https://github.com/w3c/transitions/issues/194

# Substantive changes

- Changed normative definition of the rdf:JSON datatype in §10.2 "The rdf:JSON Datatype" to describe a normative canonicalization and not relying on other datatypes. This is in response to [Issue 323](https://github.com/w3c/json-ld-syntax/issues/323). This also changed the normative language around JSON literals in the API document, hence the republication of that document, too.

TL;DR for the changes: while the previous version defined the value space of the RDF JSON literal referring to other XSD datatypes, this leads to confusion and semantic issues as noted in [Issue 323](https://github.com/w3c/json-ld-syntax/issues/323). As a result, the WG has changed the value space consisting of canonicalized JSON text, with the canonicalization rules defined by the specification itself. See [diff file](https://tinyurl.com/shuj268) for the changes.

(There has been a number of other, non-substantive changes based on implementation feedbacks, mostly improving the API document's algorithmic details)

# Any changes in normative references?

No

# Any changes in requirements?

No

# Wide Review of substantive changes

See [Issue 323](https://github.com/w3c/json-ld-syntax/issues/323)

# Issues status

See remaining issues in [Syntax](https://github.com/w3c/json-ld-syntax/issues?utf8=✓&q=is%3Aissue+is%3Aopen+-label%3Adefer-future-version) and [API](https://github.com/w3c/json-ld-api/issues?utf8=✓&q=is%3Aissue+is%3Aopen+-label%3Adefer-future-version).

# Formal Objections

None

# Any changes in implementation information?

None

# Deadline for further comments

2020-03-30

# Any changes in patent disclosures?

None

cc: @gkellogg @pchampin @dlongley @BigBlueHat @azaroth42

**Comment by swickr** [comment]
Transition approved.

**Comment by plehegar** [comment]
https://www.w3.org/TR/2020/CR-json-ld11-20200305/
https://www.w3.org/TR/2020/CR-json-ld11-api-20200305/
